compiler/test/stdlib/range.test.gr

```diff
@@ -75,3 +75,11 @@ assert results == [2, 3, 4, 5]
 Range.forEach((idx) => {
   fail "Shouldn't be called"
 }, exclusiveSameRange)
+
+let idfn = (a) => a
```

Comment:
Could we instead do something here that produces a new value? Mapping over `toString` could be a good candidate!